parsec/core/invite/organization.py

```diff
@@ -0,0 +1,65 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+from typing import Optional
+from pendulum import now as pendulum_now
+
+from parsec.crypto import SigningKey
+from parsec.api.data import UserCertificateContent, DeviceCertificateContent, UserProfile
+from parsec.api.protocol import HumanHandle, DeviceID
+from parsec.core.types import LocalDevice, BackendOrganizationAddr
+from parsec.core.local_device import generate_new_device
+from parsec.core.backend_connection import APIV1_BackendAnonymousCmds
+
+
+async def bootstrap_organization(
+    cmds: APIV1_BackendAnonymousCmds,
```

Comment:
So we're still using APIV1 in some non-legacy cases? Are those API V1 vs V2 use cases documented somewhere?

Reply:
APIV1 is still useful for administration-related stuff (e.g. organization create&status) and for organization bootstrap

The rational behind this is adiministration related stuff would be much better exposed with a REST API, so we didn't provide it as part of the APIv2. Hence we could provide a REST API in the future, then totally deprecate the APIv1.

The only part that could be discussed is the organization bootstrap process. This process should be done by a Parsec client in a pretty similar fashion that the user/device claim (i.e. an invited party connect to the backend to end up with a valid device).
So it could make sense to consider the organization bootstrap command should be provided by APIv2.
However from an implementation standpoint it's slightly different (the device/user claim doesn't use the same kind of token than the organization boostrap, the claiming process works completely differently with multiple steps where the organization bootstrap has only one) so sticking with APIv1 seems like a good idea given it simplify the codebase and allows us to consider the switch to REST API in the future.